CONTRIBUTING.md

```diff
@@ -33,8 +33,9 @@ To contribute:
     [clang-tidy](https://clang.llvm.org/extra/clang-tidy/) tools can be
     integrated into your workflow to ensure formatting and stylistic
     compliance of your changes. To avoid code formatting misalignment with
-    GitHub Actions check we recommend using 10 version of clang-format tool
-    (default version on Ubuntu 22.04).
+    the GitHub Actions check we recommend using the same version of clang-format
+    as used in the CI. See `.github/workflows/pr-code-format.yml` for the
```

Comment:
Have a link to the `.github/workflows/pr-code-format.yml` file would simplify navigation.